sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationClientLiveTests.cs

```diff
@@ -259,17 +261,7 @@ public async Task GetAndListOperations(bool useTokenCredential)
 
             DocumentModelOperationDetails operationDetails = await client.GetOperationAsync(modelOperationFromList.FirstOrDefault().OperationId);
 
-            ValidateOperationSummary(operationDetails);
```

Comment:
We can't use `ValidateOperationSummary` anymore for this object since we lost the inheritance between `Details` and `Summary`. For this reason I wrote a new `ValidateOperationDetails` method.